erpcgen/src/templates/c_coders.template

```diff
@@ -239,8 +263,9 @@ codec->write({$info.name});
 {% enddef --------------------------------------- BuiltinType %}
 
 {% def encodeBinaryType(info) -----------------  %}
-{% if source == "server" && info.size != info.maxSize %}
-erpc_assert({$info.size} <= {$info.maxSize} * sizeof({$info.mallocSizeType}));
+{% if info.sizeTemp != info.maxSize %}
+erpc_assert({% if source == "client" && info.pointerScalarTypes %}*{% endif %}{$info.size} <= {$info.maxSize} * sizeof({$info.mallocSizeType}));
+
```

Comment:
one empty newline can be removed

Reply:
I thought it is better with it, but whatever, I will do it here and in the write string